acceptance-tests/src/test/resources/features/site/publication.feature

```diff
@@ -100,3 +100,16 @@ Feature: Ensure publication page displays required fields.
             | 1.9 Dataset  |
             | 1.10 Dataset |
             | 2.0 Dataset  |
+
+    Scenario: National Statistics kite mark on National statistic information type only
+        Given I navigate to the "search" page
+        When I search for ""
+        Then I click on the "National statistics" link
+        And I can click on the "Morbi tempor euismod vehicula" link
+        And I should also see:
+            | Publication Information Types | National statistics           |
+        And I can see "National statistics" link
```

Comment:
The kite mark is just an icon not a link right?

Reply:
yes, its just an icon. I'll update the "I can see" method to also take "image" as well as link|button